src/TableColumn/Generic.php

```diff
@@ -106,7 +106,8 @@ public function setHeaderPopup($popup, $icon = 'caret square down')
         $popup->popOptions = array_merge($popup->popOptions, ['on' =>'click', 'position' => 'bottom right', 'movePopup' => false]);
         $popup->stopClickEvent = true;
 
-        if (@$_GET['__atk_reload']) {
+        $need_reload = $_GET['__atk_reload'] ?? false;
```

Comment:
cant this be done like 
`if(($_GET['__atk_reload'] ?? false)) {` ?
Would save the extra var $need_reload which doesnt seem to be used anywhere else.